### PR TITLE
Add input parameter to HelloIntelliJ program

### PR DESCRIPTION
**Pull request by FarhadPathan**
Please check

### CODE REVIEW

File: src/Helper.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+public class Helper {
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public static String getInput(String prompt) {
+        String message = scanner.nextLine();
+        System.out.print(prompt + message);
+        return null; // replace with actual input
```
Comment (DavPoghosyan) [inline src/Helper.java]: return message not null

File: src/Helper.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    }
+    public static int getIntInput(String prompt) {
+        String message = scanner.nextLine();
+        System.out.print(prompt + message);
+        return 0;
```
Comment (DavPoghosyan) [inline src/Helper.java]: dont return 0, instead return whatever inputed

**Review by DavPoghosyan — CHANGES_REQUESTED** [review]

File: src/Helper.java (hunk 1 above)
Comment (FarhadPathan) [reply src/Helper.java]: now return proper value

File: src/Helper.java (hunk 2 above)
Comment (FarhadPathan) [reply src/Helper.java]: now return number

**Review by DavPoghosyan — APPROVED** [review]

**Pull request merged**